src/systemctl/systemctl.c

```diff
@@ -114,7 +115,7 @@ bool arg_plain = false;
 bool arg_firmware_setup = false;
 usec_t arg_boot_loader_menu = USEC_INFINITY;
 const char *arg_boot_loader_entry = NULL;
-bool arg_now = false;
+bool arg_now = true;
```

Comment:
Why changed? The flag is for enabling/disabling units, and unrelated to 'show' command.